src/infra/ChatAnalyticsReporter.ts

```diff
@@ -17,12 +18,15 @@ export class ChatAnalyticsReporter {
    * when the endpoint field from config is not provided.
    */
   private readonly endpoint: string;
+  /** Whether to enable session tracking for analytics events */
+  private readonly sessionTrackingEnabled: boolean;
 
   constructor(
-    { apiKey, env, region, endpoint }: ChatAnalyticsConfig,
+    { apiKey, env, region, endpoint, sessionTrackingEnabled = true }: ChatAnalyticsConfig,
```

Comment:
might want to set it to true by default in the US and false in EU

Reply:
done!